src/pages/en/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,405 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Upgrade Guide
```

Comment:
How do you feel about changing the title here?

- more specific
- should look nice in OpenGraph images as people share this out
- I’d bet this is moderately better for SEO

```suggestion
title: Upgrade to Astro v2
```

Reply:
And while we’re here, maybe also update the title & description of the old guide now at `upgrade-to/v1`? Less clear cut what the rename should be. Maybe something like this?

```diff
-title: Migration Guide
-description: How to migrate your project to latest version of Astro.
+title: Legacy v0.x Migration Guide 
+description: Archived guide documenting changes between pre-v1 versions of Astro
```

That page also starts with

> This guide will help you migrate from older versions of Astro to the latest version.

Probably worth updating that too.